public/app/features/dashboard/dashgrid/PanelChrome.tsx

```diff
@@ -135,6 +138,28 @@ export class PanelChrome extends Component<Props, State> {
           next: (data) => this.onDataUpdate(data),
         })
     );
+
+    // TODO? check if the panel has X=time?
+    // Note these events will only be broadcast if the dashboard is enabled
+    if (this.props.plugin) {
+      this.subs.add(
+        this.props.dashboard.events.getStream(LiveDashboardTick).subscribe({
```

Comment:
Use subscribe method on the event bus instead, getStream is only if you need to use Observable operators (map / merge etc)